app/cypress/integration/upload.spec.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+/// <reference types="Cypress" />
+
+context('Homepage', () => {
+  beforeEach(function () {
+    const mockedDateString = '2022-10-10';
+    const mockedDate = new Date(mockedDateString);
+    cy.useMockedTime(mockedDate);
+    cy.sqlFixture('dev/001_intake');
+    cy.visit('/');
+  });
+
+  // Commenting out radio inputs until we pass in proper names or ids to select from
+
+  it('should start and fill the first page of the form', () => {
+    cy.get('body').happoScreenshot({ component: 'Applicant Landing Page' });
+
+    cy.get('h1').contains('Welcome');
+
+    cy.get('a').contains('program details');
+
+    // Todo: find a way around using these wait
+    cy.wait(2000);
+
+    cy.get('button').contains('Go to dashboard').click();
+
+    cy.url().should('contain', '/dashboard');
+
+    // Dashboard page
+    cy.get('h1').contains('Dashboard');
+
+    cy.get('body').happoScreenshot({ component: 'Dashboard Page' });
+
+    cy.get('a').contains('Edit').click();
+    cy.wait(2000);
+    cy.get('a').contains('Supporting documents').click();
+    cy.wait(2000);
```

Comment:
Instead of clicking its way through the application, our test could create the application via an sql fixture (we could create a `db/data/testFixtures/applications.sql` file) and do a `cy.visit('/form/1/12')`.

We should also have a `reset_db.sql` file in our fixtures, and this file should be run before every test, to avoid side effects. Looking at the happo diff, this test has a side effect (it creates an application, which shows up in the next test).

Reply:
Done